components/domains/registerCheckboxes.tsx

```diff
@@ -63,22 +71,33 @@ const RegisterCheckboxes: FunctionComponent<RegisterCheckboxes> = ({
         </div>
         {!isArOnforced ? (
           <InputHelper
-            helperText={`Enabling a subscription permits Starknet ID to renew your domain automatically every year for you! This approval gives us only the possibility to renew your domain once per year maximum (${gweiToEth(
-              ethRenewalPrice
-            )} ETH/year) and we'll cover the transaction fee for you!`}
+            helperText={`Enabling a subscription permits Starknet ID to renew your domain automatically every year for you! This approval gives us only the possibility to renew your domain once per year maximum ${
+              ethRenewalPrice ? `(${gweiToEth(ethRenewalPrice)} ETH/year)` : ""
+            } and we'll cover the transaction fee for you!`}
           >
             <div
               className="flex items-center justify-left text-xs cursor-pointer"
               onClick={onChangeRenewalBox}
             >
               <Checkbox checked={renewalBox} sx={{ padding: 0 }} />
-              <p className="ml-2 text-left flex items-center gap-2 relative">
+              <p className="ml-2 mr-10 text-left flex items-center gap-2 relative">
                 Enable subscription and don&apos;t pay gas for your yearly
                 renewal
               </p>
             </div>
           </InputHelper>
         ) : null}
+        {showMainDomainBox ? (
+          <div
+            className="flex items-center justify-left text-xs cursor-pointer"
+            onClick={onChangeMainDomainBox}
+          >
+            <Checkbox checked={mainDomainBox} sx={{ padding: 0 }} />
+            <p className="ml-2 mr-10 text-left flex items-center gap-2 relative">
+              Set {domain ?? "your domain"} as your main domain
```

Comment:
If the domain is undefined we have:

``Set your domain as your main domain`` we can just say ``Set as your main domain`` so we don't repeat ``domain``